test_data/tricky_example_fish_win_fixed_output.txt

```diff
@@ -0,0 +1,174 @@
+║   │   │   │   │  │  │  │   │ ▓ │   │   │ 3 │   │   │       ║
+║   │   │   │   │  │  │  │   │ ▓ │ 4 │   │   │   │   │       ║
+║   │   │   │   │  └──┘6 │   │ ▓ │   │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │  ┌──┐1 │ ▓ │   │   │ 2 │   │   │       ║
+║   │   │   │   │   │  │  │  │ ▓ │   │   │ 3 │   │   │       ║
+║   │   │   │   │   │  │  │  │ ▓ │ 4 │   │   │   │   │       ║
+║   │   │   │   │   │  └──┘6 │ ▓ │   │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │  ┌──┐1 │ ▓ │   │   │   │ 2 │   │       ║
+║   │   │   │   │   │  │  │  │ ▓ │   │   │ 3 │   │   │       ║
+║   │   │   │   │   │  │  │  │ ▓ │ 4 │   │   │   │   │       ║
+║   │   │   │   │   │  └──┘6 │ ▓ │   │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │  ┌──┐1 │ ▓ │   │   │   │ 2 │   │       ║
+║   │   │   │   │   │  │  │  │ ▓ │   │   │   │ 3 │   │       ║
+║   │   │   │   │   │  │  │  │ ▓ │ 4 │   │   │   │   │       ║
+║   │   │   │   │   │  └──┘6 │ ▓ │   │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │   │  ┌──┐1 │   │   │   │ 2 │   │       ║
+║   │   │   │   │   │   │  │  │  │   │   │   │ 3 │   │       ║
+║   │   │   │   │   │   │  │  │  │ 4 │   │   │   │   │       ║
+║   │   │   │   │   │   │  └──┘6 │   │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │   │  ┌──┐1 │   │   │   │   │ 2 │       ║
+║   │   │   │   │   │   │  │  │  │   │   │   │ 3 │   │       ║
+║   │   │   │   │   │   │  │  │  │ 4 │   │   │   │   │       ║
+║   │   │   │   │   │   │  └──┘6 │   │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │   │  ┌──┐1 │   │   │   │   │ 2 │       ║
+║   │   │   │   │   │   │  │  │  │   │   │   │   │ 3 │       ║
+║   │   │   │   │   │   │  │  │  │ 4 │   │   │   │   │       ║
+║   │   │   │   │   │   │  └──┘6 │   │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══╧═══════╝
+╔═══╤═══╤═══╤═══╤═══╤═══╤═══╤════════╤═══╤═══╤═══╤═══╤═══════╗
+║   │   │   │   │   │   │   │  ┌──┐1 │   │   │   │ 2 │       ║
+║   │   │   │   │   │   │   │  │  │  │   │   │   │ 3 │       ║
+║   │   │   │   │   │   │   │  │4 │  │   │   │   │   │       ║
+║   │   │   │   │   │   │   │  └──┘6 │   │   │ 5 │   │       ║
+╚═══╧═══╧═══╧═══╧═══╧═══╧═══╧════════╧═══╧═══╧═══╧═══╧═══════╝
```

Comment:
Those and the following lines are wrong. the central tile always misses the ▓ characters